scrapers/utils/bs/bs.py

```diff
@@ -0,0 +1,47 @@
+import argparse
+import logging
+
+logging.basicConfig(level=logging.ERROR)
+
+import spacy
+
+
+with open("utils/bs/numbers.txt", "r") as txtfile:
+    numbermapping = {l.strip(): str(i) for i, l in enumerate(txtfile.readlines())}
+numbermapping["hundert"] = "100"
+numbermapping["tausend"] = "1000"
+numbermapping["einen"] = "1"
+numbermapping["eine"] = "1"
+numbermapping["eins"] = "1"
+
+
+def replace_numberwords(text: str) -> str:
+    return " ".join([numbermapping.get(word.strip(".,:;"), word) for word in text.split(" ")])
+
+
+def preprocess(text: str) -> str:
+    text = text.replace(",", "").replace("\n", " ")
+    return replace_numberwords(text)
+
+
+def parse(text: str) -> dict:
+    result = {}
+    nlp = spacy.load("utils/bs/nlp")
+    doc = nlp(text)
+    for sent in doc.sents:
+        sdoc = nlp(sent.string.strip())
+        for ent in sdoc.ents:
+            try:
+                result[ent.label_] = int(ent.text.strip('.,'))
+            except ValueError:
+                logging.warning(f"Error parsing: {ent.text.strip('.,')}")
+    return result
+
+
+def update_indirect_numbers(result: dict) -> dict:
+    if result.get("NUMCUL_CONF", None) and result.get("NUMCUL_CONF_RESIDENTS"):
```

Comment:
You don't need to pass `None` explicitly, it is a default for the `default` argument:

Help on built-in function get:

```
get(key, default=None, /) method of builtins.dict instance
    Return the value for key if key is in the dictionary, else default.
```

Reply:
```python
if "NUMCUL_CONF" in result and "NUMCUL_CONF_RESIDENTS" in result:
```
nice! i did not know that!

Reply:
let me add that, its way more readable than my mess :)

Reply:
Actually in python you would generally not use something like `if "NUMCUL_CONF" in result`, it follows the EAFP coding style.
But I agree: passing None does not make sense.